src/main/scala/com/resy/BookReservationWorkflow.scala

```diff
@@ -95,9 +95,14 @@ object BookReservationWorkflow extends StrictLogging {
         (v \ "config" \ "type").asOpt[String],
         (v \ "config" \ "token").asOpt[String]
       )}
+      .tap(listBookingTypes)
```

Comment:
Grab the list of dining-types available; not a comprehensive list as it's only those available for booking, but will help with future accuracy of sniping hopefully.